Web/starter_project_barbies/pages/team/Team.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import TeamsImage from '../../public/teams_page_img.svg'
+import Image from 'next/image'
+import TeamMemberCard from "@/components/team/TeamMemberCard";
+import { TeamMemberCardProps } from "@/types/team/TeamMemberCardProps";
+
+const DummyDataGenerator = (): TeamMemberCardProps[] => {
+  const data:TeamMemberCardProps[] = []
```

Comment:
see I don't imagine it would be a good idea to have a list of props
I think if you have a TeamMember object defined separately (like mentioned in the other PR #29) you can have your data be a list of that